extensions/standard-processors/processors/ListenSyslog.h

```diff
@@ -45,13 +45,15 @@ class ListenSyslog : public NetworkListenerProcessor {
   EXTENSIONAPI static const core::Property MaxBatchSize;
   EXTENSIONAPI static const core::Property ParseMessages;
   EXTENSIONAPI static const core::Property MaxQueueSize;
+  EXTENSIONAPI static const core::Property SSLContextService;
```

Comment:
I think we could  `EXTENSIONAPI static const core::Property ClientAuth;` here aswell.

Reply:
Updated in d0e2ac6c3abeff8d46f8e15fd80816dd86fa9b13